### PR TITLE
Ryan dev4 - 2023-12-21

### PR DESCRIPTION
**Pull request by ryanwooa**
- Update leftside bar CSS and function
- Add 'My projects' page
- Add 'NewProject' widget

### CODE REVIEW

**Review by kesesek — COMMENTED** [review]
Where did you find these icons? I'm looking for an arrow icon.  No other problems.

**Comment by ryanwooa** [comment]
> Where did you find these icons? I'm looking for an arrow icon. No other problems.

@kesesek 
image
I download them in Todoist. 
Click F12 in your browser, you can inspect the element in the page. Click the button at 1, you could choose the element in the page. Step 2, click where you want to know, the browser will locate the element in the right widow as shown in the picture with number 3. Then you could copy them.

File: styles/scss/utils/_variable.scss [hunk 1]
```diff
@@ -36,19 +36,43 @@ $itemMinHeightMd: 30rem;
...
+$borderRadiusSm: 0.6rem;
+$hoverColor: #f6efee;
+$selectedColor: #fee6e3;
 
 $mobileWidth: 28rem;
 $tabletWidth: 42rem;
 $appSideBarWidth: 30.5rem;
 $minComputerWidth: 62.5rem;
-$maxListWidth: 80rem;
+$maxListWidth: 91.47rem;
```
Comment (zli213) [inline styles/scss/utils/_variable.scss:47]: Why use non-integer values? I noticed that even when converted to pixels, the values are not integers. What is the purpose of this?

**Review by zli213 — CHANGES_REQUESTED** [review]
Please check

File: styles/scss/utils/_variable.scss (hunk 1 above)
Comment (ryanwooa) [reply styles/scss/utils/_variable.scss]: I want 800px, but it couldn't be expelled, so I took an approximation.
Now I change it to 800px.

**Pull request merged**